README.md

```diff
@@ -1320,6 +1320,8 @@ OIDC flow is required for [Social Login][].
     ]
     ```
 
+    The following IdPs are supported: `ADOBE`, `AMAZON`, `APPLE`, `DISCORD`, `FACEBOOK`, `GITHUB`, `GITLAB`, `GOOGLE`, `LINE`, `LINKEDIN`, `MICROSOFT`, `ORCID`, `PAYPAL`, `PAYPAL_SANDBOX`, `QUICKBOOKS`, `SALESFORCE`, `SPOTIFY`, `XERO`, `YAHOO`, and `YAHOOJP`.
```

Comment:
I noticed that Smart Card IdP is marked as self service. Should you clarify that these will only be available through self service EA? https://github.com/okta/okta-signin-widget#smart-card-idp